### PR TITLE
Release 8.2.0

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+## [8.2.0](https://github.com/theforeman/puppet-puppet/tree/8.2.0) (2018-01-25)
+[Full Changelog](https://github.com/theforeman/puppet-puppet/compare/8.1.0...8.2.0)
+
+**Implemented enhancements:**
+
+- Make max-queued-requests and max-retry-delay configurable [\#569](https://github.com/theforeman/puppet-puppet/issues/569)
+- add compile\_mode parameter to puppetserver.conf [\#574](https://github.com/theforeman/puppet-puppet/pull/574) ([miksercz](https://github.com/miksercz))
+- Make performance tuning defaults more safe [\#572](https://github.com/theforeman/puppet-puppet/pull/572) ([kasimon](https://github.com/kasimon))
+- Add `server\_max\_queued\_requests` and `server\_max\_retry\_delay` parameters [\#570](https://github.com/theforeman/puppet-puppet/pull/570) ([baurmatt](https://github.com/baurmatt))
```
Comment (mmoll) [inline CHANGELOG.md]: with the backticks, the underscores are not needed

Comment (ekohl) [reply CHANGELOG.md]: https://github.com/skywinder/github-changelog-generator generated that. Looks like they need a bit smart markdown handling.

Comment (ekohl) [reply CHANGELOG.md]: (updated btw)

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**